### DISCUSSION TITLE
[Feature] Manually tag person in photo.

### DISCUSSION THREAD

**Discussion by spusuf** [immich-app/immich#3834] [Feature Request]
### The feature

Sometimes it is good to have the option to manually tag a person who is either:
Slightly out of the photo (example dress shots, edge of photo shots),
Obscured in the photo (turned around or behind people in a group photo),
Or not even in the photo (taking a photo of an activity/movie/coffee shop you went with a person). 

These would all be nice examples on when you still want these photos to come up when you open the person's album, but was not detected by machine learning for the above reasons.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #2472

**Comment by Buxxter** [comment]
Hello there!
It's an original feature request not mentioned in #2472 
No opened tickets with same proposals at this time. Can we reopen it or need to copy/paste in new one?

**Comment by lopaced** [comment]
This feature would be very useful and appears to be highly anticipated by many users (myself included).

Unfortunately, several tickets (this one, #5333, #7301, #9810, ...) have been closed as duplicates of ticket #2472.

Now that ticket #2472 is closed, the feature described in the current ticket is still not implemented.

In my opinion, reopening this ticket would be a relevant step to address this issue
